cmake/Codegen.cmake

```diff
@@ -76,18 +76,23 @@ if(INTERN_BUILD_ATEN_OPS)
 
   file(GLOB_RECURSE all_python "${CMAKE_CURRENT_LIST_DIR}/../torchgen/*.py")
 
-  # RowwiseScaled.cu requires sm90a flags
+  # RowwiseScaled.cu requires sm89/sm90a flags
   if(USE_CUDA)
     set(ROWWISE_SCALED_MM_FILE "${CMAKE_CURRENT_LIST_DIR}/../aten/src/ATen/native/cuda/RowwiseScaledMM.cu")
 
     # Get existing arch flags
     torch_cuda_get_nvcc_gencode_flag(EXISTING_ARCH_FLAGS)
 
     # Check NVCC version and existing arch flags
-    if(CMAKE_CUDA_COMPILER_VERSION VERSION_GREATER_EQUAL 12.0 AND
-      EXISTING_ARCH_FLAGS MATCHES ".*compute_90.*")
-      set_source_files_properties(${ROWWISE_SCALED_MM_FILE}
-        PROPERTIES COMPILE_FLAGS "-gencode arch=compute_90a,code=sm_90a")
+    if(CMAKE_CUDA_COMPILER_VERSION VERSION_GREATER_EQUAL 12.0)
+      if(EXISTING_ARCH_FLAGS MATCHES ".*compute_86.*")
+        set_source_files_properties(${ROWWISE_SCALED_MM_FILE}
+          PROPERTIES COMPILE_FLAGS "-gencode arch=compute_89,code=sm_89")
```

Comment:
Probably an interesting followup, is to enable sm89 codegen for more float8 ops (though I'm not sure if native conversion operations will be faster if compiled for sm89)